src/app/components/MediaLoader/index.styles.ts

```diff
@@ -50,10 +50,9 @@ export default {
       !isEmbedded && commonMarginSpacing,
     ],
 
-  audioMediaContainer: ({ palette }: Theme) =>
+  audioMediaContainer: () =>
     css({
-      backgroundColor: palette.WHITE,
-      height: '165px',
+      height: '130px',
```

Comment:
Worth just double checking the Live Radio and OD-Audio/Podcast pages as they will share the same styling for their audio players.